Marlin/src/lcd/marlinui.h

```diff
@@ -305,6 +305,11 @@ class MarlinUI {
     static void pause_print();
     static void resume_print();
 
+    #if ENABLED(DGUS_LCD_UI_MKS)
+      static void pause_print_move();
```

Comment:
Why this only for this display? 🤔

Reply:
Because this pause action is related to the coordinates set on the screen. Of course, it's not only applicable to this screen. The reason for using the macro is that people can easily recognize that this function is added for H43. If you need to use this function for other purposes, you can also use it.

Reply:
Marlin already has a parking feature, so there is no need to add another one. Let's try to refactor this so that it uses the existing methods.

Reply:
Yes, it sounds like a good proposal, and I will continue to let it be implemented as much as possible using the existing methods.